app/view_models/admin/needs_attention_auctions_view_model.rb

```diff
@@ -12,7 +12,7 @@ def drafts_partial
   end
 
   def drafts
-    @_drafts ||= Auction.unpublished.map do |auction|
+    @_drafts ||= AuctionQuery.new.unpublished.map do |auction|
```

Comment:
sure, I do think using `Auction` rather than `AuctionQuery` would be more consistent, but I defer to you